+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu May 25 21:55:22 2023 - INFO  - Start to run test.
+ dmesg
+ grep -iE 'error|fail|not support|no such'
+ grep -vE 'Failed to initialise from firmware|platform does not support|virgl 3d acceleration not supported| _OSC failed| fail to add MMCONFIG information|res=success|CPU model not supported|not support BPF/cgroup firewalling'
[    0.714331] syscon-poweroff: probe of poweroff failed with error -16
[    0.716741] riscv-pmu-sbi: Perf sampling/filtering is not supported as sscof extension is not available
+ CHECK_RESULT 0 1 0 'Dmesg has false information.'
+ actual_result=0
+ expect_result=1
+ mode=0
+ error_log='Dmesg has false information.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 1x
+ test -n 'Dmesg has false information.'
+ LOG_ERROR 'Dmesg has false information.'
+ message='Dmesg has false information.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Dmesg has false information.'
Thu May 25 21:55:23 2023 - ERROR - Dmesg has false information.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dmesg_messages_log.sh line 24'
+ message='oe_test_dmesg_messages_log.sh line 24'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dmesg_messages_log.sh line 24'
Thu May 25 21:55:24 2023 - ERROR - oe_test_dmesg_messages_log.sh line 24
+ '[' 0 -eq 1 ']'
+ return 0
+ grep -iE 'error|fail|not support|no such' /var/log/messages
+ grep -vE 'Failed to initialise|platform does not support|Couldn'\''t write '\''fq_codel'\''|Failed to init entropy source|KD_FONT_OP_GET failed while|virgl 3d acceleration|ignorelockingfailure|not support BPF/cgroup firewalling| /etc/lvm/backup/openeuler: stat failed:|Failed to read VG openeuler |ovsdb: Could not connect:|secret-key: failure to generate|dbus-org.freedesktop.resolve1.service|imjournal: No statefile exists|net.ipv4.icmp_ignore_bogus_error_responses|pam_faillock|rasdaemon: wait_access() failed|wait_access\(\) failed|rasdaemon: Can'\''t get|lm_sensors.service|Failed to start Hardware Monitoring Sensors|unit=lm_sensors|/etc/lvm/backup/openeuler_openeuler|package at does not exist| _OSC failed| fail to add MMCONFIG information|failed to open file /etc/ndctl/keys/nvdimm-master|/etc/samba/secrets.tdb\) No such file or directory|Unable to watch \(/root/.ssh/*|ipv6: duplicate address check failed for|linklocal6: failed to generate an address|res=success|Can'\''t open PID file /run/restorecond.pid|CPU model not supported|\[hwrng \]: Initialization Failed|\[rndr  \]: Initialization Failed'
grep: /var/log/messages: No such file or directory
+ CHECK_RESULT 1 1 0 'Dmesg has false information.'
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log='Dmesg has false information.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ CASE_RESULT 0
+ case_re=0
+ test -z 1
+ test 1 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu May 25 21:55:24 2023 - ERROR - failed to execute the case.
+ exit 1
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_dmesg_messages_log.sh post_test'
+ message='oe_test_dmesg_messages_log.sh post_test'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'oe_test_dmesg_messages_log.sh post_test'
Thu May 25 21:55:25 2023 - INFO  - oe_test_dmesg_messages_log.sh post_test
